api/types/types.proto

```diff
@@ -481,6 +481,12 @@ message ClusterNameSpecV2 {
     string ClusterName = 1 [ (gogoproto.jsontag) = "cluster_name" ];
 }
 
+// ConfigResourceRequest is a request relating to a configuration resource.
+message ConfigResourceRequest {
+    // SkipValidation is used to skip JSON schema validation.
+    bool SkipValidation = 1 [ (gogoproto.jsontag) = "skip_validation" ];
```

Comment:
If we decide that we need to pass skip validation from the client, I think we should do it using `"google.golang.org/grpc/metadata"`. This will be more extendable to other options and won't bloat our client with options. We could add helper functions that work on either `ctx` or `client` to add these options, similar to `WithDialOptions` and `WithCallOptions`.

Reply:
I removed `SkipValidation` from here so the empty request is now sufficient.

However you can keep these notes in mind since `Get{App,Database}Servers` already have ` bool SkipValidation` in their request messages.